ios/FluentUI.Demo/FluentUI.Demo/Demos/BadgeViewDemoController.swift

```diff
@@ -23,10 +24,16 @@ class BadgeViewDemoController: DemoController {
         return badge
     }
 
-    func addBadgeSection(title: String, style: BadgeView.Style, isEnabled: Bool = true) {
+    func addBadgeSection(title: String, style: BadgeView.Style, isEnabled: Bool = true, overrideColor: Bool = false) {
         addTitle(text: title)
         for size in BadgeView.Size.allCases.reversed() {
             let badge = createBadge(text: "Kat Larrson", style: style, size: size, isEnabled: isEnabled)
+            if overrideColor {
+                badge.backgroundColor = Colors.Palette.blueMagenta20.color
+                badge.backgroundSelectedColor = Colors.Palette.cyanBlue20.color
```

Comment:
Is this building correctly for you?
The failing checks have this on the log:

            FluentUIFramework.initializeAppearance()
                              ^
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift:16:31: note: use 'initializeAppearance(with:whenContainedInInstancesOf:)' instead
            FluentUIFramework.initializeAppearance()
                              ^~~~~~~~~~~~~~~~~~~~
                              initializeAppearance
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/BadgeViewDemoController.swift:33:23: error: value of type 'BadgeView' has no member 'backgroundSelectedColor'
                badge.backgroundSelectedColor = Colors.Palette.cyanBlue20.color
                ~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~
/Users/runner/work/fluentui-apple/fluentui-apple/ios/FluentUI.Demo/FluentUI.Demo/Demos/BadgeViewDemoController.swift:35:23: error: value of type 'BadgeView' has no member 'labelSelectedTextColor'
                badge.labelSelectedTextColor = Colors.Palette.gray100.color
                ~~~~~ ^~~~~~~~~~~~~~~~~~~~~~

** BUILD FAILED **

Reply:
yes, but this isn't related to my change. I just rebase from my master to see if there are additional changes affecting this.